docs/checkers/eslint.md

```diff
@@ -37,3 +37,4 @@ Advanced object configuration table of `options.eslint`
 | lintCommand        | `string`                                                                                                   | This value is required | `lintCommand` will be executed at build mode, and will also be used as default config for dev mode when `eslint.dev.eslint` is nullable.                                                                                 |
 | dev.overrideConfig | [`ESLint.Options`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/eslint/index.d.ts) | `undefined`            | **(Only in dev mode)** You can override the options of the translated from `lintCommand`. Config priority: `const eslint = new ESLint({cwd: root, ...translatedOptions, ...pluginConfig.eslint.dev?.overrideConfig, })`. |
 | dev.logLevel       | `('error' \| 'warning')[]`                                                                                 | `['error', 'warning']` | **(Only in dev mode)** Which level of ESLint should be emitted to terminal and overlay in dev mode                                                                                                                       |
+debounceTime | `number` | `undefined` | Avoid repeated formatting files when your editor set lint on save action,and avoid multiple check in a short time |     
```

Comment:
```suggestion
dev.debounce | `number` | `undefined` | Avoid repeated formatting files when your editor set lint on save action, and avoid multiple checks in a short time |     
```

Should be put in the `dev` filed as we do not need it in build mode.

Reply:
i will chanage option name as your suggestion.thx